pkgs/test_core/lib/src/runner/suite.dart

```diff
@@ -149,6 +150,11 @@ final class SuiteConfiguration {
   final bool? _ignoreTimeouts;
   bool get ignoreTimeouts => _ignoreTimeouts ?? false;
 
+  /// The timeout for loading a test suite.
+  final Timeout? _suiteLoadTimeout;
+  Timeout get suiteLoadTimeout =>
+      _suiteLoadTimeout ?? const Timeout(Duration(minutes: 12));
```

Comment:
I implemented `--suite-load-timeout` the same way as `--ignore-timeouts` was implemented. However, it's not optimal that we have two locations for the default value. Here and in `pkgs/test_core/lib/src/runner/configuration/args.dart`. Should we improve this? Or not? Or just writing a comment to update the default value also in the other file?